### ISSUE TITLE
Docker file is not idempotent

### ISSUE THREAD

**Issue by thedug** [dockerfile/nodejs#10]
The docker file downloads "latest" from node repository. This can change  overtime, but the hash won't change. This means that your outcome will be dependent on the state of your cache and when you run the command. The node version would ideally be included in that RUN command.

**Comment by jtmarmon** [comment]
yes pls. just spent a long time trying to figure out why my package was working on my machine but not in the container

**Comment by ivanoats** [comment]
:+1: I would like to be able to specify latest 0.10 - similar to how nvm works. 0.10 is really the most stable one still.

**Comment by notslang** [comment]
seems like https://github.com/nodejs/docker-node has tags figured out
